app/extensions/brave/content/scripts/pageInformation.js

```diff
@@ -59,6 +59,14 @@
           expr['arguments'].forEach((argument) => { args.push(traverse(argument)) })
           return value[traverse(op1.property, true)].apply(value, args)
 
+        case 'NewExpression':
+          op1 = expr.callee
+          if (op1.type !== 'Identifier') throw new Error('unexpected callee: ' + op1.type)
+          if (op1.name !== 'Set') throw new Error('only new Set(...) is allowed, not new ' + op1.name)
+          args = []
+          expr['arguments'].forEach((argument) => { args.push(traverse(argument)) })
+          return new Set(args[0])
```

Comment:
Don't you want `return new Set(...args)` for full generality? No point collecting the  traversed arguments in args past the first one if you use only the first one.